### DISCUSSION TITLE
CloudlogCat

### DISCUSSION THREAD

**Discussion by ChrisL79** [magicbug/Cloudlog#2814] [Aurora]
**This is for reporting bugs with the code, if this issue is regarding setting up on your local server please use the forum see readme for details**

**Describe the bug**
Trying to install CloudlogCat on Windows 10 x64. On opening the program, there is an Axtive X error. Pressing continue opens the program. Then input the server address and api. The program will not connect to server.

**To Reproduce**
Steps to reproduce the behaviour:

1. Install CloudlogCat in W10 x64

**Expected behaviour**
For the program to connect to my instance. 

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by phl0** [comment]
Does anything appear in your Cloudlog log files?

**Comment by ChrisL79** [comment]
Negative

**Comment by magicbug** [comment]
Have you installed Omnirig? do you see frequency info in cloudlogcat even with it not connecting to the server, can we see a screenshot of settings and the main window.  Remember to blank out your API key

Also really need to know what the >  Axtive X error is

**Comment by ChrisL79** [comment]
Thanks for the reply. 

I will double check all of that tomorrow and post what i find. 

Thank you!

**Comment by magicbug** [comment]
How did you get on?

**Comment by ChrisL79** [comment]
Thanks for getting back to me.

I have Covid...AGAIN, which is making concentrating on things that bit harder.

I have had somewhat success with Aurora.

The issue i am having now is Aurora seems to be timing out if i do not make quick successive qsos. Any more than a minute without logging anything and its like Aurora goes to sleep. The only way to fix it is to restart Aurora.

Any idea?

Chris.

**Comment by magicbug** [comment]
We're still referring to CloudlogCAT per the topic? 

I'd not expect Aurora to go to sleep for any reason

**Comment by magicbug** [comment]
Converted this to a discussion as its not related to code issue with cloudlog itself
